README.md

```diff
@@ -77,6 +79,35 @@ Here's an example of using it in nixos configuration.
 }
 ```
 
+### Example: A `devShell` Flake
+
+Assuming you have a valid `shell.nix`, running `nix develop` will create a shell with the default nightly Rust toolchain installed:
+
+```nix
+{
+  description = "Blah";
```

Comment:
maybe write something like 'devShell example'

Reply:
How about 
"Using `devShell` and `nix develop`"

?

Reply:
Sounds much better!

Reply:
Done! Thanks for reviewing